pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -295,6 +295,11 @@ stdenv.mkDerivation {
       hardening_unsupported_flags+=" stackprotector fortify pie pic"
     ''
 
+    + optionalString targetPlatform.isWasm ''
+      echo "--allow-undefined --no-entry" >> $out/nix-support/cc-ldflags
```

Comment:
I don't think you want this. You can define a "foo.imports" file next to your library files that the linker will read for allowed undefined symbols (which should specifically just be the syscall symbols and users' symbols).

I don't know why you'd have --no-entry

Reply:
Yeah this may only be needed for compiler-rt. It complains about missing _start symbols otherwise.